cmd/pulumi-test-language/providers/output_only_invoke_provider.go

```diff
@@ -84,7 +86,7 @@ func (p *OutputOnlyInvokeProvider) GetSchema(
 		},
 		Functions: map[string]schema.FunctionSpec{
 			"output-only-invoke:index:myInvoke": {
-				OutputStyleOnly: true,
+				Plain: &bfalse,
```

Comment:
Optional nit: I tend to use a `ref` helper function so it reads better: `func ref[T any](v T) *T { return &v }`. This would then be `ref(false)`.

I think this functionality is being added to `new` in Go 1.26.